### PR TITLE
Add sendHasBrand param to draft-annotations-api request

### PR DESCRIPTION
**Pull request by mchompalova**

### CODE REVIEW

File: annotations/annotations_rw_client_test.go [hunk 1]
```diff
@@ -19,6 +20,7 @@ import (
 )
 
 var testingClient = fthttp.NewClientWithDefaultTimeout("PAC", "test-annotations-publisher")
+var draftsURL = "/drafts/content/:uuid/annotations"
```
Comment (ivan-p-nikolov) [inline annotations/annotations_rw_client_test.go]: To minimize the possibility of test dependencies we should probably use `const` more often.
```suggestion
const draftsURL = "/drafts/content/:uuid/annotations"
```

File: resources/publish.go [hunk 2]
```diff
@@ -53,9 +53,9 @@ func Publish(publisher annotations.Publisher, httpTimeOut time.Duration) func(w
 		if fromStore {
 			publishFromStore(ctx, publisher, uuid, w)
 		} else {
-			json.Unmarshal(bodyBytes, &body)
+			err := json.Unmarshal(bodyBytes, &body)
```
Comment (ivan-p-nikolov) [inline resources/publish.go]: We don't need to create a new err object here. We can reuse the one in the scope already.

File: resources/publish.go [hunk 3]
```diff
@@ -53,9 +53,9 @@ func Publish(publisher annotations.Publisher, httpTimeOut time.Duration) func(w
 		if fromStore {
 			publishFromStore(ctx, publisher, uuid, w)
 		} else {
```
Comment (ivan-p-nikolov) [inline resources/publish.go]: We can `return` in the if body and remove the `else` to lower the complexity of the code.

File: README.md [hunk 4]
```diff
@@ -60,9 +60,17 @@ curl http://localhost:8080/__health | jq
...
+```
+curl http://localhost:8080/draft/content/b7b871f6-8a89-11e4-8e24-00144feabdc0/annotations/publish?fromStore=true -XPOST
+```
+
+А POST request with fromSource=true retrieves the latest annotations from draft-annotations-api, persists them to the PAC database draft and published-annotations tables, and then publishes them to UPP.
+
 ####Publish with Body####
 
-This endpoint saves the provided annotations to the PAC database (within the draft and published annotations collections), and will subsequently publish the annotations to UPP.
+This endpoint does the same as Publish from Store with an extra step - it saves in PAC the annotations provided in the body before getting the latest annotations from draft-annotations-api.
```
Comment (karsov) [inline README.md]: This sounds confusing what happen first.

File: annotations/annotations_rw_client_test.go [hunk 5]
```diff
@@ -35,8 +37,8 @@ func TestAnnotationsRWGTGFails(t *testing.T) {
 	server := mockGtgServer(t, false)
 	defer server.Close()
 
-	client, err := NewAnnotationsClient(server.URL+"/%s", testingClient)
-	err = client.GTG()
+	client, _ := NewAnnotationsClient(server.URL+"/%s", testingClient)
```
[the remark this answers is not in the document]
Comment (mchompalova) [reply annotations/annotations_rw_client_test.go]: Because I'm getting a linter exception.

[the remark this answers is not in the document]
Comment (mchompalova) [reply annotations/annotations_rw_client_test.go]: Because the linter complains :)

File: resources/publish.go (hunk 3 above)
Comment (mchompalova) [reply resources/publish.go]: Fixed.

File: README.md (hunk 4 above)
Comment (mchompalova) [reply README.md]: I rewrote it.

**Pull request merged**